astro.config.ts

```diff
@@ -36,7 +36,37 @@ const createSROnlyLabel = (text: string) => {
 // https://astro.build/config
 export default defineConfig({
 	site: 'https://docs.astro.build/',
-	integrations: [preact(), react(), sitemap(), astroAsides()],
+	integrations: [
+		preact(),
+		react(),
+		sitemap({
+			i18n: {
+				defaultLocale: 'en',
+				locales: {
+					ar: 'ar',
+					bg: 'bg',
+					bn: 'bn',
+					da: 'da',
+					de: 'de',
+					en: 'en-US',
+					es: 'es-ES',
+					fi: 'fi',
+					fr: 'fr-CA',
+					hu: 'hu',
+					it: 'it',
+					ja: 'ja',
+					kr: 'kr',
+					nl: 'nl',
+					pl: 'pl',
+					'pt-BR': 'br-PT',
+					ru: 'ru',
+					'zh-CN': 'zh-CN',
+					'zh-TW': 'zh-TW',
+				},
```

Comment:
Sarah is right, but I think we can actually avoid duplicating this data here completely.

Instead, we can import our existing dictionary of languages:

```js
import languages from './src/i18n/languages';
```

And then generate the `locales` object from that:

```suggestion
				locales: Object.fromEntries(Object.keys(languages).map((lang) => [lang, lang])),
```